crowbar_framework/app/models/nova_service.rb

```diff
@@ -199,7 +202,7 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
             node.crowbar["crowbar"]["network"]["nova_fixed"]["use_vlan"] = false
             node.save
           end
-        end
+	end
```

Comment:
> Same here.

Done in https://github.com/crowbar/barclamp-nova/commit/a1841c17f6a764b84e31838f1b4c06f28ee887b3